docs/understanding-your-pipeline/data-products/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Data Products Introduction"
+sidebar_position: 7
+sidebar_label: "🆕 Data Products Introduction"
+description: "Snowplow's data products enable organizations to easily generate AI and BI-ready data that is reliable, clear, compliant, accurate, and predictable"
+---
+
+Data products are Snowplow’s solution to helping organisations more easily create & democratise data that is AI & BI ready; data that is reliable, explanatory, compliant, accurate, and predictable. 
+
+With data products, you can:
+
+* Set clear ownership for the data being created
+* Make tracking implementation easier
+* Deliver better governance around your data
+* More easily communicate what the data means and how to use it 
+* Collaborate more effectively with the various teams involved in delivering value from your data
+
+## What is a Data Product
+
+A data product is a well documented dataset. By documenting what data you are collecting, where, the meaning of that data and how to use it, you can break down the barriers that exist today between the many teams involved in the data value chain (i.e. from those implementing the tracking to those analysing the data). 
+
+At its core, a data product has:
+
+* An explicit owner; that is responsible for maintaining and updating the data over time
+* Consumers; who use the data to deliver use cases and are impacted by upstream changes 
+
+Data products at Snowplow are underpinned by the concept of a data contract. They provide an upfront agreement on what to expect from the data being collected and capture changes to that dataset over time.
```

Comment:
Change 'they provide ...' to this instead: 

They act as a formal agreement between the producers of data products and the consumers of data products, and support better collaboration around the data being created.

Reply:
done